app/code/Magento/Eav/Model/Config.php

```diff
@@ -174,6 +182,7 @@ public function __construct(
         \Magento\Framework\Validator\UniversalFactory $universalFactory,
         SerializerInterface $serializer = null,
         ScopeConfigInterface $scopeConfig = null,
+        StoreManagerInterface $storeManager = null,
```

Comment:
Could you please move it to the last position?

Reply:
Oh sure, also optional parameter